attachment-viewer/build.gradle

```diff
@@ -49,12 +50,12 @@ dependencies {
 
     implementation 'com.github.chrisbanes:PhotoView:2.3.0'
 
-    implementation 'io.reactivex.rxjava2:rxkotlin:2.4.0'
-    implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
+    implementation libs.rx.rxKotlin
+    implementation libs.rx.rxAndroid
 
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}"
     implementation 'androidx.core:core-ktx:1.6.0'
-    implementation 'androidx.appcompat:appcompat:1.3.1'
+    implementation libs.androidx.appCompat
     implementation "androidx.recyclerview:recyclerview:1.2.1"
 
     implementation 'com.google.android.material:material:1.4.0'
```

Comment:
this lib is also on several modules

Reply:
Yes, there are a lot of dependencies that should be moved there, I just moved the basic ones for feedback. I will continue with the others.